pkg/apis/cluster/mutation/mutation_test.go

```diff
@@ -123,3 +123,30 @@ func TestStandardizeClusterResourceModels(t *testing.T) {
 		}
 	}
 }
+
+func TestSetDefaultClusterResourceModels(t *testing.T) {
+	type args struct {
+		cluster *clusterapis.Cluster
+	}
+	tests := []struct {
+		name string
+		args args
+	}{
+		{
+			name: "test set default Cluster",
+			args: args{
+				cluster: &clusterapis.Cluster{},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			SetDefaultClusterResourceModels(tt.args.cluster)
+		})
+		modelCounts := len(tt.args.cluster.Spec.ResourceModels)
+		if modelCounts != 9 {
+			t.Errorf("expected cluster.Spec.ResourceModels count is 9, but get %d", modelCounts)
+		}
+
```

Comment:
This blank line maybe needs to remove, the lint reports this check.

Reply:
ok